util/timer/src/timer.rs

```diff
@@ -61,12 +62,12 @@ impl TimerLoop {
         }
     }
 
-    pub fn new_timer(&self, name: TimerName) -> TimerApi {
+    pub fn new_timer(&self) -> TimerApi {
```

Comment:
How about adding a new constructor that crates unnamed timer?

Reply:
There's no use case for it.

Reply:
https://github.com/CodeChain-io/codechain/pull/1121/commits/5a9a58137f5532d034eacbc290e4a512b2405ff8#diff-22a0b3dcef5d27fab89655ebdb330967R95
reseal_timer can have a name on construction.

Reply:
I forgot it. Sorry. I've added it.